conda-forge.yml

```
@@ -6,3 +6,8 @@ azure:
 # qt 5.12 claims to not support ppc64le. It likely needs patching
 # to get it through the build process
 provider: {linux_aarch64: default}
```

```suggestion
provider: {linux_aarch64: azure}
```

It would be great to build this in CI rather than having to manually build it on the conda-forge ARM server.